operator/internal/manifests/mutate.go

```diff
@@ -121,6 +121,7 @@ func mergeWithOverride(dst, src interface{}) error {
 
 func mutateConfigMap(existing, desired *corev1.ConfigMap) {
 	existing.BinaryData = desired.BinaryData
+	existing.Data = desired.Data
```

Comment:
Do you know why `Data` was omitted in the original code? This seems like a subtle but significant change...

Reply:
We use to store everything in `BinaryData`, i.e. the loki config, the tenant overrides, etc. because there was the fear that Loki might introduced config options that are non-UTF8. This makes reading configuration a bit harder because we need to pipe everything into a bas64 decoder. I opted here in to make the rules configmap easier to read.